docs/r.rst

```diff
@@ -1,24 +1,38 @@
 Redirection
 -----------
 
+..
+   Include a named paragraph in the page, where the javascript code below will
+   place any message.
 
-.. raw:: HTML
-
-   <html>
-     <head>
-        <p id="demo"></p>
-
-        <script>
-        let redirects = new Map([
-            ['demo', 'url1'],
-            ['GIN', 'basics/101-139-gin.html']
-        ]);
-        redirect = redirects.get(window.location.href.replace(/.*\?/, ""));
-        if (redirect == undefined) {
-          document.getElementById("demo").innerHTML = "no idea"
-        } else {
-          window.location.replace(window.location.href.replace(/r.html\?.*/, redirect))
-        }
-        </script>
-     </head>
-   </html>
+.. raw:: html
+
+   <p id="message"></p>
```

Comment:
This could have a default message that anticipates a successful redirection, like "You will be redirected". Maybe we even render the target URL here to (ala, if this doesn't work, click here).

Reply:
yes. agreed